modules/ServerAPI/src/client/SaltBrowser.html

```diff
@@ -109,84 +108,163 @@ <h5>Searching for Salts containing text "<i class="bv_exptSearchTerm"></i>"</h5>
     <!-- CREATE SALT SECTION -->
     <div class="modal hide bv_createSalt">
         <div class="modal-header">
-            <h3>Create Salt</h3>
+            <h3>Register New Salt</h3>
         </div>
         <div class="modal-body">
+            <!-- Input Salt Section-->
             <div class="bv_chemicalStructureForm">
             </div>
+            <p><i>Use the sketcher to sepcify the molecular structure of the salt. Pay close attention to changes as they will be used in calculating Lot Molecular Weight. Use the + and - buttons to add positive or negative charges</i></p>
+            <br>
+            <br>
             <label for="abbrevName">Abbreviation: </label><br>
             <input type="text" id="abbrevName" name="abbrevName" class="bv_abbrevName"><br>
             <label for="saltName">Name:</label><br>
             <input type="text" id="saltName" name="saltName" class="bv_saltName">
-            <div class="bv_createStatusIndicator hide">
-                <table width="100%"></table>
-                <tr>
-                    <td style="text-align: center; width: 100%">
-                        <h4>Creating Salt</h4>
-                        <div class="progress progress-striped active" style="width: 100%;">
-                            <div class="bar" style="width: 100%;"></div>
-                        </div>
-                    </td>
-                </tr>
-            </div>
-            <div class="bv_saltCreatedSuccessfullyMessage hide">
-                <p>Salt <span class="bv_saltCodeName"></span> has successfully been created</p>
-            </div>
-            <div class="bv_errorCreatingSaltMessage hide">
-                <p>An error occured creating Salt <span class="bv_saltCodeName"></span>.  Please contact support for additional help.</p>
-            </div>
         </div>
         <div class="modal-footer">
 		    <span class="bv_createButtons">
-		      <a href="#" class="btn btn-primary bv_cancelCreate">Cancel</a>
-		      <a href="#" class="btn btn-danger bv_confirmCreateSaltButton">Create Salt</a>
+		      <a href="#" class="btn btn-primary bv_cancelCreate">Back</a>
+		      <a href="#" class="btn btn-danger bv_confirmCreateSaltButton">Next</a>
 			  </span>
-		    <span class="bv_okayButton hide">
-			    <a href="#" class="btn btn-primary bv_cancelCreate">Okay</a>
+		    <span class="bv_okayCreateButton hide">
+			    <a href="#" class="btn btn-primary bv_okayCreate">Okay</a>
 		    </span>
         </div>
     </div>
+    <!-- Confirmation of Values Section -->
+    <div class="modal hide bv_confirmCreateSalt">
```

Comment:
Looks like this part is still a "modal" / popup. I'd like to structure this more like the Stereo Category or Vendor browsers, where the "create" view replaces the browser.